packages/commonwealth/client/scripts/helpers/localStorage.ts

```diff
@@ -0,0 +1,39 @@
+const KEY_REFCODE = 'common-refcode';
+const REFCODE_EXPIRATION_DAYS = 7;
+
+export const getLocalStorageRefcode = () => {
```

Comment:
can we make these generic, ex: 
```
enum LocalStorageKeys {
ReferralCode = 'ReferralCode'
}

export const get = (key: LocalStorageKeys) => {
if(key === /*any key specific branch case*/) {
}

return localStorage.getItem(key);

}

// usage
localstorage.get(LocalStorageKeys.ReferralCode)
```

this way we can migrate existing local storage logic to this helper.

Reply:
I modified it a bit to make it more generic